src/style.css

```diff
@@ -1 +1,46 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+.header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 50px;
+  font-family: "Roboto", sans-serif;
+}
+
+.nav {
+  padding: 10px 50px;
+}
+
+.ul {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 50px;
+}
+
+.li {
+  height: 100%;
+  display: block;
+  font-style: normal;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 500px;
+  padding-left: 20px;
+}
+
+.a {
```

Comment:
check class names, you shouldn't use tag names as class names, it should be meaningful and understandable
also, I don't see styles for active link, check the task description one more time

Reply:
I'll check everything and make it better, thanks for commenting :)